site-manage/site-group-manager/src/main/webapp/WEB-INF/templates/fragments/javascript.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+<body>
+<script th:fragment="indexJs" th:inline="javascript">
+  /*<![CDATA[*/
+  $('#delete-groups-cancel-button').click(function(e) {
+    e.preventDefault;
+    window.location = window.location.href.replace('/sakai-site-group-manager.helper', '');
+  });
+
+  function checkSubmitButton() {
```

Comment:
You need to namespace these functions - they're in the window scope and may clash with other tools. You could just use groupmanager.checkSubmitButton, or something like that.

Reply:
Ok, I'll work on that, gotta learn how to do it!